### DISCUSSION TITLE
Feature Request: Provide a way to show the type information for a binding on hover

### DISCUSSION THREAD

**Discussion by njlr** [ionide/ionide-vscode-fsharp#1432] [Ideas]
**Is your feature request related to a problem? Please describe.**

*feature request*

When I hover over a binding, I can see the _name_ of its type. Usually, I would like to see more information about the type. For example, for a record I would like to see the members of the record. 

**Describe the solution you'd like**

I would like the ability to hover over the type name in the pop-up for a binding, which would show another pop-up with the type information. 

So: 

 * When I hover over binding... 
 * Ionide shows a pop-up with its type name...
 * And when I hover over the type name in that pop-up...
 * Ionide shows an additional pop-up with the type information

**Describe alternatives you've considered**

Currently I have to `Right Click -> Go to Type Definition`, but then I have navigated away from the code I am working on. 

**Additional context**

None

[a comment by an automated account is not shown]

**Comment by cartermp** [comment]
This is a core tooling behavior in the compiler service itself, where bindings corresponding to a record just shows the name of the record. This is consistent with object definitions as well.

![image](https://user-images.githubusercontent.com/6309070/98583345-0d08cd80-2279-11eb-9e4f-0cfec09a507c.png)

I suppose this is a call for maintainers of Ionide to see if they want to deviate from the core FCS behavior or not.
